File: Easy/Java/2500. Delete Greatest Value in Each Row.java
class Solution {
    public int deleteGreatestValue(int[][] grid) {
        int sum = 0;

        for(int[] i : grid){
            Arrays.sort(i);
        }

        for(int i = 0; i < grid[0].length; i++){
            int max = 0;
            for(int[] e : grid){
                if(max <= e[i])
                    max = e[i];
            }
            sum += max;
        }

        return sum;
    }
}

File: Easy/JavaScript/2500. Delete Greatest Value in Each Row.js
var deleteGreatestValue = function(grid) {
    let sum = 0;

    grid.forEach((i) => {
        i.sort((a, b) => a - b);
    });

    for(let i = 0; i < grid[0].length; i++){
        let max = 0;
        grid.forEach((e) => {
            if(max <= e[i])
                max = e[i];
        });
        sum += max;
    };

    return sum;
};
